client/python/datajunction/models.py

```diff
@@ -42,6 +42,14 @@ class NodeStatus(str, enum.Enum):
     INVALID = "invalid"
 
 
+class NodeType(str, enum.Enum):
+    """
+    DJ node types
+    """
+
+    METRIC = "metric"
```

Comment:
For completeness I think we should add all node types to this enum. In the function where `models.NodeType.METRIC.value` there's also `"dimension"` there that could be changed to `models.NodeType.DIMENSION.value` for consistency. I also think it's reasonable to just hardcode the values and get rid of this enum all together, only because the client code is pretty thin.